test/common_test/joint_transmitted_wrench_features.cc

```diff
@@ -431,5 +454,13 @@ TYPED_TEST(JointTransmittedWrenchFixture, ContactForces)
   EXPECT_TRUE(gz::physics::test::Equal(expectedWrenchAtMotorJointInJoint,
                                        wrenchAtMotorJointInJoint, 1e-4));
 }
+*/
```

Comment:
I'd prefer not to disable this test for all physics engines if it passes for some of them